spark/src/main/scala/geotrellis/spark/io/json/package.scala

```diff
@@ -49,19 +50,20 @@ package object json {
     def write(metaData: RasterMetaData) = 
       JsObject(
         "cellType" -> metaData.cellType.toJson,
-        "extent" -> metaData.extent.toJson,
-        "crs" -> metaData.crs.toJson,
-        "tileLayout" -> metaData.tileLayout.toJson
+        "dataExtent" -> metaData.dataExtent.toJson,
+        "layoutExtent" -> metaData.layoutExtent.toJson,
+        "tileLayout" -> metaData.tileLayout.toJson,
```

Comment:
Did we not end up making a type that is the composition of a `TileLayout` and `Extent`? If that were it's own type, then we wouldn't have to use the `dataExtent` name, which feels kind of awkward for me.